src/main/java/org/openpnp/gui/JobPanel.java

```diff
@@ -651,8 +651,10 @@ public void jobStart() throws Exception {
         if (title.equals("Solder Paste")) {
             jobProcessor = Configuration.get().getMachine().getPasteDispenseJobProcessor();
         }
-        else if (title.equals("Pick and Place")) {
-            jobProcessor = Configuration.get().getMachine().getPnpJobProcessor();
```

Comment:
Correct me if I am wrong, but it looks like you removed the PnpJobProcessor entirely.